.travis.yml

```diff
@@ -4,16 +4,16 @@ jobs:
     - os: osx
       osx_image: xcode11.3
       script:
-        - cd build
-        - cmake -DBINARY_RELEASE=ON ..
-        - sudo cmake --build . --target package -j $(sysctl -n hw.physicalcpu)
+        - cmake -S. -Bbuild -DBINARY_RELEASE=ON
+        - sudo cmake --build build --target package --config Release -j $(sysctl -n hw.physicalcpu)
     - os: linux
       dist: bionic
       arch: ppc64le
       script:
-        - cd build
-        - cmake -DBINARY_RELEASE=ON ..
-        - cmake --build . --target package
+        # CMake 3.10: no -S/-B and -j is build tool specific.
+        - mkdir build && cd build
+        - cmake .. -DBINARY_RELEASE=ON
+        - cmake --build . --target package -- -j $(nproc)
```

Comment:
oh that might explain the slow builds :)

Reply:
5 minutes of it are still `sudo apt install`

Reply:
Still from 9min to 5min is a good improvment for a oneline change :)